deps/sqlite3.gyp

```diff
@@ -112,8 +118,9 @@
           "copies": [
             {
               "files": [
-                '<(SHARED_INTERMEDIATE_DIR)/sqlcipher-amalgamation-<@(sqlite_version)/>(openssl_root)/libeay32.dll',
-                '<(SHARED_INTERMEDIATE_DIR)/sqlcipher-amalgamation-<@(sqlite_version)/>(openssl_root)/msvcr120.dll'
+                '<(SHARED_INTERMEDIATE_DIR)/sqlcipher-amalgamation-<@(sqlite_version)/>(openssl_root)/libssl.lib',
+                '<(SHARED_INTERMEDIATE_DIR)/sqlcipher-amalgamation-<@(sqlite_version)/>(openssl_root)/libcrypto.lib',
+                '<(SHARED_INTERMEDIATE_DIR)/sqlcipher-amalgamation-<@(sqlite_version)/>(openssl_root)/ossl_static.pdb'
```

Comment:
Is this needed, or can we remove this entire action?

AFAIK only the dll's needed to be copied, and the .lib files should now be statically linked as part of the .node file.

Reply:
Can be removed indeed, done!